README.md

```diff
@@ -461,9 +461,16 @@ docker run --rm \
        ghcr.io/ther1d/shell_gpt --chat rainbow "what are the colors of a rainbow"
 ```
 
+When using a container, please note:
+* The \[E\]xecute option for --shell with interaction will not work, since it would try this Execute in the docker container.  
+=> setting the `SHELL_INTERACTION` environment variable to false , makes sense.
+* Since, most likely the os and shell of your container are not identical to the environment you want help with:  
+set the environment variables `OVERWRITE_OS_NAME` and `OVERWRITE_SHELL_NAME` according to your setup.
+
+
 Example of a conversation, using an alias and the `OPENAI_API_KEY` environment variable:
 ```shell
-alias sgpt="docker run --rm --env OPENAI_API_KEY --volume gpt-cache:/tmp/shell_gpt ghcr.io/ther1d/shell_gpt"
+alias sgpt="docker run --rm --env OPENAI_API_KEY --env SHELL_INTERACTION=false OVERWRITE_OS_NAME=Debian --env OVERWRITE_SHELL_NAME=bash --volume gpt-cache:/tmp/shell_gpt ghcr.io/ther1d/shell_gpt"
```

Comment:
```suggestion
alias sgpt="docker run --rm --volume gpt-cache:/tmp/shell_gpt --env OVERWRITE_OS_NAME=$(uname -s) --env OVERWRITE_SHELL_NAME=$(echo $SHELL) ghcr.io/ther1d/shell_gpt "
```

Reply:
Nice example!
I made a commit to use the pretty OS name instead of "Linux":
$(grep -oP '^PRETTY_NAME="\K[^"]+' /etc/os-release)

Reply:
Let's use `uname -s` as suggested above, since **/etc/os-release** does not exist on macOS. Given that we are supporting both Linux and macOS, ShellGPT should function correctly on both platforms.